caddy.go

```diff
@@ -836,7 +839,7 @@ var (
 	instancesMu sync.Mutex
 )
 
-const (
+var (
```

Comment:
> Even so, the default is *still* Caddyfile even with CoreDNS compiled in? I just
change it in *my* startup....?

If you change it only when the DNS server is being _used_, I guess that's okay. How are you doing that?